labelbox/schema/organization.py

```diff
@@ -126,6 +129,40 @@ def remove_user(self, user: "User") -> None:
             updateUser(where: {id: $%s}, data: {deleted: true}) { id deleted }
         }""" % (user_id_param, user_id_param), {user_id_param: user.uid})
 
+    def create_resource_tag(self, tag=None) -> ResourceTag:
+        """
+        Creates a resource tag.
+            >>> tag = {'text': 'tag-1',  'color': 'ffffff'}
+
+        Args:
+            tag (dict): A resource tag {'text': 'tag-1', 'color': 'fffff'}
+        Returns:
+            The created resource tag.
+        """
+        tag_text_param = "text"
+        tag_color_param = "color"
+
+        query_str = """mutation CreateResourceTagPyApi($text:String!,$color:String!) {
+                createResourceTag(input:{text:$%s,color:$%s}) {%s}}
+        """ % (tag_text_param, tag_color_param,
+               query.results_query_part(ResourceTag))
+
+        params = {
+            tag_text_param: tag.get("text", ""),
+            tag_color_param: tag.get("color", "")
+        }
+        res = self.client.execute(query_str, params)
+        return ResourceTag(self.client, res['createResourceTag'])
+
+    def get_resource_tags(self) -> List[ResourceTag]:
+        """
+        Returns all resource tags for an organization
+        """
+        query_str = """query GetOrganizationResourceTagsPyApi{organization{resourceTag{%s}}}""" % (
+            query.results_query_part(ResourceTag))
+
+        return self.client.execute(query_str)['organization']['resourceTag']
```

Comment:
This should return an iterable of ResourceTag objects instead of json. Ideally this would be a paginated collection. At the very lest it should do the following:
```return [ResourceTag(self.client, tag) for tag in self.client.execute(query_str)['organization']['resourceTag']]```